src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -100,7 +100,7 @@ def load_config(self):
         else:
             self.metadata_mgr = MetadataManager(self.fs, self.config_path, 0o640)
 
-    def set_attrs(self, path, size, isolate_namespace, pool, uid, gid):
+    def set_attrs(self, path, size, isolate_namespace, pool, uid, gid, namespace=None):
```

Comment:
Is there a need to pass in an additional parameter? Can't that be inferred by passing a non-None namespace?

Reply:
Meaning, we use `isolate_namespace` itself to pass the value or generate one instead? Or, we generate one as needed before the call to `set_attrs` and then pass it along in `namespace` and remove `isolate_namespace`?

Reply:
just pass namespace -- if it's invalid (None), it need not be set.

Reply:
Addressed.